projects/fmcjesdadc1/common/fmcjesdadc1_bd.tcl

```diff
@@ -1,4 +1,12 @@
 
+# RX parameters for each converter
```

Comment:
Here we should define the parameters of the merged links or  per link but not mix them. 

Per total the system has 4 lanes and 4 converters. 

If you have RX_NUM_OF_LANES = 4 also the RX_NUM_OF_CONVERTERS should be 4